api/src/colors.py

```diff
@@ -0,0 +1,15 @@
+colors = [
```

Comment:
I think these should be dataclasses for two reasons:
1. To differentiate RGB from HSV and HSB
2. So you can customize the way it's transformed into JSON. Tuples will serialize as a list when serializing into JSON. Lists are hard to maintain in a backwards compatible way and are unnecessary when you know the exact length of the thing you're returning.